vendor/assets/javascripts/webauthn-json.js

```diff
@@ -0,0 +1,184 @@
+// webauthn-json v0.3.6
+//
+// Source: https://github.com/github/webauthn-json/tree/v0.3.1
+// License: MIT - https://github.com/github/webauthn-json/blob/v0.3.1/LICENSE.md
+//
+// Generated by checking out @github/webauthn-json source code and running
+// $ ROLLUP_WATCH=1 yarn build
```

Comment:
@lgarron Hi, thanks for webauthn-json!

Had to run the above to include @github/webauthn-json here.
Would you recommend something different?

Reply:
The *recommended* approach is to use `npm`, and we expect that most projects will use a build system.

If not, you can get a browser (UMD) build using:

    npm install @github/webauthn-json
    cat node_modules/@github/webauthn-json/dist/index.js

If you include this in a file on a page the old-school way, you should be able to access `webauthnJSON` (e.g. on `window`).

Your approach generates an equivalent un-minified file. If you want the un-minified file and don't care about code size, then that's probably the best way to do it.

Reply:
Hi @lgarron,

Thanks for the response!
We're good for now this way then.

Reply:
> The _recommended_ approach is to use `npm`, and we expect that most projects will use a build system.

Yep. Opened the following (https://github.com/rubygems/rubygems.org/issues/2126) while ago, just because of this.